scheduler/src/cook/kubernetes/api.clj

```diff
@@ -142,12 +142,14 @@
   "Initialize the pod watch. This fills all-pods-atom with data and invokes the callback on pod changes."
   [^ApiClient api-client compute-cluster-name all-pods-atom cook-pod-callback]
   ; We'll iterate trying to connect to k8s until the initialize-pod-watch-helper returns a watch function.
-  (let [tmpfn (fn []
+  (let [{:keys [reconnect-delay-ms]} (config/kubernetes)
+        tmpfn (fn []
                 (try
                   (initialize-pod-watch-helper api-client compute-cluster-name all-pods-atom cook-pod-callback)
                   (catch Exception e
                     (log/error e "Error during initial setup of looking at pods for" compute-cluster-name)
-                    (Thread/sleep (get-in config/config [:settings :kubernetes :reconnect-delay-ms]))
+                    (log/info "Sleeping" reconnect-delay-ms "milliseconds before reconnect")
```

Comment:
Can we add the compute cluster name to this log?

Reply:
Ok.